app-admin/passwordsafe/passwordsafe-1.16.0.ebuild

```diff
@@ -0,0 +1,89 @@
+# Copyright 2023 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+WX_GTK_VER="3.0-gtk3"
+
+inherit cmake desktop flag-o-matic optfeature wxwidgets xdg
+
+MY_PV="${PV/_beta/BETA}"
+DESCRIPTION="Password manager with wxGTK based frontend"
+HOMEPAGE="https://pwsafe.org/ https://github.com/pwsafe/pwsafe/"
+SRC_URI="https://github.com/pwsafe/pwsafe/archive/${MY_PV}.tar.gz -> ${P}.tar.gz"
+S="${WORKDIR}/pwsafe-${MY_PV}"
+
+LICENSE="Artistic-2"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="qr test +xml yubikey"
+RESTRICT="!test? ( test )"
+
+DEPEND="
+	dev-libs/openssl:0=
+	net-misc/curl
+	sys-apps/file
+	sys-apps/util-linux
+	x11-libs/libXt
+	x11-libs/libXtst
+	x11-libs/wxGTK:${WX_GTK_VER}[X]
+	qr? ( media-gfx/qrencode )
+	xml? ( dev-libs/xerces-c )
+	yubikey? ( sys-auth/ykpers )"
+RDEPEND="${DEPEND}"
+BDEPEND="${DEPEND}
```

Comment:
can you confirm all "${DEPEND}" are fall into BDEPEND? I haven't checked myself, but just ask..

Reply:
Maybe.
I will do a complete re-evaluation of the dependencies in a VM and call back with the result,

Reply:
I tested the ebuild in a fresh deployed VM with minimal package set. I removed some dependencies which are already in the system set. All other DEPENDs are also BDEPENDs, that's correct.

Reply:
quote part of the message from [1] 
Starting with EAPI 7, build dependencies are split into two variables: BDEPEND and DEPEND. BDEPEND specifies dependencies applicable to CBUILD, i.e. programs that need to be executed during the build, e.g. virtual/pkgconfig. DEPEND specifies dependencies for CHOST, i.e. packages that need to be found on built system, e.g. libraries and headers. 

I think, starting from EAPI-7,  the DEPEND is actually split into two parts - DEPEND and BDEPEND. Packages from BDEPEND actually need to run (execute) at build time, like generating data which will be used by the package in run-time; while packages from DEPEND is normally libraries and header files which only contribute to compiling or linking phases, but do not need to run at CBUILD environment.

[1] https://devmanual.gentoo.org/general-concepts/dependencies/index.html